root/edit/details/AddAnnotation.js

```diff
@@ -21,7 +21,7 @@ type AddAnnotationEditT = {
   +display_data: {
     +changelog: string,
     +entity_type: AnnotatedEntityTypeT,
-    [AnnotatedEntityTypeT]: AnnotatedEntityT,
+    [annotated_entity: AnnotatedEntityTypeT]: AnnotatedEntityT,
```

Comment:
Since this is only for documentation, we can be consistent and use always camelCase (so annotatedEntity). Same for many others below :) 

Also, I would expect that this is actually an entity type, not a specific entity, looking at the flow typing.